hw1_interview_questions/avishai_nudelman.ts

```diff
@@ -0,0 +1,25 @@
+/**
+     We can rotate digits by 180 degrees to form new digits. 
+     When 0, 1, 6, 8, 9 are rotated 180 degrees, they become 0, 1, 9, 8, 6 respectively. 
+     When 2, 3, 4, 5, and 7 are rotated 180 degrees, they become invalid. 
+     A confusing number is a number that when rotated 180 degrees becomes a different number with each digit valid.
```

Comment:
what about the number
69?

It's not a confusing number, but you count it as confusing.

I would first define a helper method that checks if a number is confusing, and unit test that.
Then you would have discovered your mistake.